core/templates/components/summary-tile/topic-summary-tile.component.ts

```diff
@@ -80,6 +87,12 @@ export class TopicSummaryTileComponent {
 
     return '#' + newRValue + newGValue + newBValue;
   }
+
+  // Remove this method when translation service is extended.
```

Comment:
For any "future work", always add a TODO comment with an issue number. Other comments are meant to stay in the codebase long-term.